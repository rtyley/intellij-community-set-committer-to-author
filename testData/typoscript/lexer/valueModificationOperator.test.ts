value1:=addToList(4,5)
value2:=addToList(4,5
value3:=( 4 and some text   )
value4:
value5:=add To My List ( 4,5)
value6:=add To My List(4,5)  some ignored text
value7: addToList( ,5)