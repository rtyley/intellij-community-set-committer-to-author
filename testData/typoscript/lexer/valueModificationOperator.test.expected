OBJECT_PATH_ENTITY ('value1')
MODIFY (':=')
MODIFICATION_FUNCTION ('addToList')
MODIFICATION_OPEN ('(')
MODIFICATION_VALUE ('4,5')
MODIFICATION_CLOSE (')')
WHITE_SPACE ('
')
OBJECT_PATH_ENTITY ('value2')
MODIFY (':=')
MODIFICATION_FUNCTION ('addToList')
MODIFICATION_OPEN ('(')
MODIFICATION_VALUE ('4,5')
WHITE_SPACE ('
')
OBJECT_PATH_ENTITY ('value3')
MODIFY (':=')
MODIFICATION_OPEN ('(')
WHITE_SPACE (' ')
MODIFICATION_VALUE ('4 and some text')
WHITE_SPACE ('   ')
MODIFICATION_CLOSE (')')
WHITE_SPACE ('
')
OBJECT_PATH_ENTITY ('value4')
BAD_CHARACTER (':')
WHITE_SPACE ('
')
OBJECT_PATH_ENTITY ('value5')
MODIFY (':=')
MODIFICATION_FUNCTION ('add To My List')
WHITE_SPACE (' ')
MODIFICATION_OPEN ('(')
WHITE_SPACE (' ')
MODIFICATION_VALUE ('4,5')
MODIFICATION_CLOSE (')')
WHITE_SPACE ('
')
OBJECT_PATH_ENTITY ('value6')
MODIFY (':=')
MODIFICATION_FUNCTION ('add To My List')
MODIFICATION_OPEN ('(')
MODIFICATION_VALUE ('4,5')
MODIFICATION_CLOSE (')')
IGNORED_TEXT ('  some IGNORED_TEXT')
WHITE_SPACE ('
')
OBJECT_PATH_ENTITY ('value7')
BAD_CHARACTER (':')
WHITE_SPACE (' ')
OBJECT_PATH_ENTITY ('addToList')
MULTILINE_OPEN ('(')
IGNORED_TEXT (' ,5)')